projects/examples/src/app/components/action-menu/with-separators/action-menu-with-separators.example.component.html

```diff
@@ -12,6 +12,7 @@
         [actions]="actions"
         [actionDisplayConfig]="actionDisplayConfig"
         [dropdownTriggerBtnText]="'vcd.cc.action.menu.actions'"
+        [selectedEntities]="[{}]"
```

Comment:
Why did you need to add this?

Reply:
Without selected entities, contextual actions will not be shown and in this example, we need the contextual actions to show the menu with separators.